### PR TITLE
Added support for s390x

### PR DESCRIPTION
**Pull request by guirish**
## Description
Added support for s390x. The routereflector changes w.r.t s390x has been tested and verified with calico v3.0.3 and v3.04 and all ST's are passing.

### CODE REVIEW

**Comment by chakravarthy392** [comment]
@fasaxc  could you kindly review above changes.

**Comment by fasaxc** [comment]
Sorry, missed this one, will add it to my queue

File: Dockerfile-s390x [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+# The Dockerfile-s390x is copied from routereflector/Dockerfile.
+# Modifications done includes:
+# 1) Base image has been changed from ubuntu:14.04 to s390x/ubuntu:16.04
+# 2) Added python3-minimal dependency
+
+FROM s390x/ubuntu:16.04
+
+RUN apt-get update && \
+        apt-get install -y python3-minimal
```
Comment (fasaxc) [inline Dockerfile-s390x]: Please can you add `&& rm -rf /var/lib/apt/lists/*` to this command.  It leaves a cleaner/smaller image.

File: Dockerfile-s390x [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# For details and docs - see https://github.com/phusion/baseimage-docker#getting_started
+
+# The Dockerfile-s390x is copied from routereflector/Dockerfile.
+# Modifications done includes:
+# 1) Base image has been changed from ubuntu:14.04 to s390x/ubuntu:16.04
```
Comment (fasaxc) [inline Dockerfile-s390x:19]: Why the change to the base image?

**Review by fasaxc — CHANGES_REQUESTED** [review]
Looks good, just a couple of questions.

File: Dockerfile-s390x (hunk 2 above)
Comment (guirish) [reply Dockerfile-s390x:19]: Hi @fasaxc, there is no published s390x/Ubuntu:14.04 images on docker hub, hence we have used s390x/Ubuntu:16.04

File: Dockerfile-s390x (hunk 1 above)
Comment (guirish) [reply Dockerfile-s390x]: @fasaxc, we will incorporate said change. Thanks.

File: Dockerfile-s390x (hunk 2 above)
Comment (fasaxc) [reply Dockerfile-s390x:19]: OK.  FYI We're keen to switch to alpine at some point for image size but, for now, there's no alpine bird image.

**Review by fasaxc — APPROVED** [review]
